react-stopwatch-component/dist/styles.css

```diff
@@ -0,0 +1,23 @@
+.circle {
+  border: 5px solid black;
+  border-radius: 50%;
+  width: 100px;
+  height: 100px;
+}
+
+h1 {
+  text-align: center;
+  position: relative;
+  top: 10px;
+}
+.fas {
+  position: relative;
+  left: 50px;
+  top: 25px;
+  cursor: pointer;
+}
```

Comment:
You don't want to add CSS rules to an existing CSS class from a CSS library. This can cause issues in other parts of your app and will cause confusion with other developers working on the project because the classes won't behave as expected or as described in the CSS libraries documentation. A better solution is to make your own class and add it to the elements that need it:

```CSS
.icon {
  position: relative;
  left: 50px;
  top: 25px;
  cursor: pointer;
}
```